java/de/yessoft/android/mohammad/model/NewsItem.java

```diff
@@ -0,0 +1,87 @@
+package de.yessoft.android.mohammad.model;
+
+import android.util.ArrayMap;
+import android.util.Log;
+
+import com.google.firebase.firestore.DocumentSnapshot;
+
+import java.util.List;
+import java.util.Map;
+
+public class NewsItem {
+    private static final String TAG = "ML_NewsItem";
+
+    private static final String KEY_HEADING = "heading";
+    private static final String KEY_IMG = "img";
+    private static final String KEY_SUMMERY = "summery";
+    private static final String KEY_FULL_NEWS_ID = "full_news_id";
+
+    private String Heading;
+    private String Summery;
+    private String Img;
+    private String FullNewsId;
+
+    // region Construct
+    private NewsItem() {
+    }
+
+    public NewsItem(DocumentSnapshot documentSnapshot) {
+        if ((String) documentSnapshot.get(NewsItem.KEY_HEADING) != null) {
+            Heading = (String) documentSnapshot.get(NewsItem.KEY_HEADING);
+        } else {
+            Log.d(TAG, "NewsItem: Error, No Heading!");
+            return;
+        }
+        Summery = (String) documentSnapshot.get(NewsItem.KEY_SUMMERY);
+        Img = (String) documentSnapshot.get(NewsItem.KEY_IMG);
+        FullNewsId = (String) documentSnapshot.get(NewsItem.KEY_FULL_NEWS_ID);
+    }
+    // endregion
```

Comment:
this should be removed

Reply:
Yes, this is what is left from the last implemetation, which means the code should be cleaned up, Maybe in the next sprint improving this thing :)